src/Models/BaseElement.php

```diff
@@ -1066,28 +1061,13 @@ public static function getBlockConfig()
         return [];
     }
 
-    /**
-     * The block actions is an associative array available for providing data to the client side to be used to describe
-     * actions that may be performed. This is available as a plain "ObjectType" in the GraphQL schema.
-     *
-     * By default the only action is "edit" which is simply the URL where the block may be edited.
-     *
-     * To modify the actions, either use the extension point or overload the `provideBlockSchema` method.
-     *
-     * @internal This API may change in future. Treat this as a `final` method.
-     * @return array
-     */
-    public function getBlockSchema()
+    public function getBlockSchema(): array
```

Comment:
Please make sure there's a PHPDoc that explains what it's for. All public and protected methods should have a PHPDoc description.

Reply:
Done

Reply:
There's a lot of information in the old doc block that's been removed. Was it actually incorrect?
It seems like we're losing a lot of context here.

Reply:
Incorrect information